src/components/Header/Header.js

```diff
@@ -0,0 +1,206 @@
+      >
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+          <title>Slack</title>
+          <path d="M7,20.11a3.28,3.28,0,1,1-3.31-3.25H7Z" fill="#e01e5a" />
+          <path
+            d="M8.62,20.11a3.28,3.28,0,1,1,6.56,0h0v8.21a3.28,3.28,0,1,1-6.55.36,2.16,2.16,0,0,1,0-.36Z"
+            fill="#e01e5a"
+          />
+          <path d="M11.89,7a3.28,3.28,0,1,1,3.29-3.28V7Z" fill="#36c5f0" />
+          <path
+            d="M11.89,8.62a3.28,3.28,0,1,1,.36,6.55H3.68a3.28,3.28,0,1,1-.36-6.55h8.57Z"
+            fill="#36c5f0"
+          />
+          <path d="M25,11.89a3.28,3.28,0,1,1,3.31,3.29H25Z" fill="#2eb67d" />
+          <path
+            d="M23.41,11.89a3.28,3.28,0,1,1-6.55.36,2.16,2.16,0,0,1,0-.36V3.68a3.28,3.28,0,1,1,6.55-.36,2.16,2.16,0,0,1,0,.36Z"
+            fill="#2eb67d"
+          />
+          <path d="M20.11,25a3.28,3.28,0,1,1-3.28,3.28h0V25Z" fill="#ecb22e" />
+          <path
+            d="M20.11,23.41a3.28,3.28,0,1,1-.36-6.55h8.57a3.28,3.28,0,1,1,.36,6.55,2.16,2.16,0,0,1-.36,0Z"
+            fill="#ecb22e"
+          />
+        </svg>
+      </a>
+      <a
+        href="https://github.com/adeo/mozaic-design-system"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+          <title>Github icon</title>
+          <path
+            d="M16.05.38A16,16,0,0,0,11,31.48c.81.14,1.09-.36,1.09-.78V28c-4.46,1-5.4-2.15-5.4-2.15A4.27,4.27,0,0,0,4.92,23.5c-1.44-1,.12-1,.12-1a3.38,3.38,0,0,1,2.44,1.65,3.41,3.41,0,0,0,4.63,1.34h0a3.36,3.36,0,0,1,1-2.13C9.55,23,5.83,21.59,5.83,15.53a6.17,6.17,0,0,1,1.64-4.28A5.8,5.8,0,0,1,7.62,7S9,6.6,12,8.66a15.25,15.25,0,0,1,8,0C23.05,6.6,24.38,7,24.38,7a5.78,5.78,0,0,1,.2,4.19,6.16,6.16,0,0,1,1.63,4.28c0,6.13-3.73,7.47-7.28,7.84A3.75,3.75,0,0,1,20,26.27v4.38c0,.52.29.92,1.1.78a16,16,0,0,0-5.06-31"
+            fill="#221e1b"
+          />
+        </svg>
+      </a>
+      <a
+        href="https://adeo.github.io/mozaic-vue/"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <svg viewBox="0 0 261.76 226.69" xmlns="http://www.w3.org/2000/svg">
+          <title>Vue.js</title>
+          <path
+            d="M161.096.001l-30.224 52.35L100.647.002H-.005L130.872 226.69 261.749 0z"
+            fill="#41b883"
+          />
+          <path
+            d="M161.096.001l-30.224 52.35L100.647.002H52.346l78.526 136.01L209.398.001z"
+            fill="#34495e"
+          />
+        </svg>
+      </a>
+    </HeaderTop>
+    <div className="ml-flexy ml-flexy--gutter">
+      <HeaderMain className="ml-flexy__col ml-flexy__col--push-1of12@from-l">
+        <HeaderTitle>Welcome to Mozaic Design System</HeaderTitle>
+        <HeaderDesc>
+          A design system to reinforce team’s delivery time, improve quality of
+          deliverables and thus help set up a coherent customer experience.
+        </HeaderDesc>
+      </HeaderMain>
```

Comment:
can we have more blank space here? We have one empty columns between the text and the pattern previews on the design

Reply:
fixed